src/Caster.Api/Features/Workspaces/Requests/Create.cs

```diff
@@ -95,6 +100,30 @@ public override async Task<Workspace> HandleRequest(Command request, Cancellatio
 
                 dbContext.Workspaces.Add(workspace);
                 await dbContext.SaveChangesAsync(cancellationToken);
+                var directoryMetrics = await dbContext.Directories.Select(d => new
```

Comment:
Might be worth putting this in an Event Handler (https://github.com/cmu-sei/Caster.Api/tree/development/src/Caster.Api/Features/Workspaces/EventHandlers) so that it triggers if a Workspace is created/deleted through another path down the road.

Reply:
Yep, Projects, too.  I like having the metric code in one place.  Great idea!